jpo-conflictmonitor/src/main/java/us/dot/its/jpo/conflictmonitor/KafkaConfiguration.java

```diff
@@ -94,12 +106,53 @@ public KafkaAdmin.NewTopics createKafkaTopics() {
             }
         }
 
-        
+        // List out existing topics
+        Admin adminClient = Admin.create(properties.createStreamProperties("ConflictMonitorAdminClient"));
+        ListTopicsOptions listTopicsOptions = new ListTopicsOptions().listInternal(true);
+        ListTopicsResult topicsResult = adminClient.listTopics(listTopicsOptions);
+        KafkaFuture<Set<String>> topicsFuture = topicsResult.names();
+        try {
+            List<String> topicNames = new ArrayList<>();
+            for(String topicName: topicsFuture.get()){
+                logger.info("Found Topic: " + topicName);
+                topicNames.add(topicName);
+            }
+
+        } catch (InterruptedException e) {
+            e.printStackTrace();
```

Comment:
I'd recommend using logger.error() instead of printing to stdout so that logging can be configured in a consistent way.

Reply:
As of Jan 28, 2024. All scans will fail regardless of the flag. This will be corrected when we update the CM to the latest version.